libmscore/style.cpp

```diff
@@ -2140,7 +2158,29 @@ void MStyle::precomputeValues()
 
 bool MStyle::isDefault(Sid idx) const
       {
-      return value(idx) == MScore::baseStyle().value(idx);
```

Comment:
Why? Does it fix an existing bug? If it is, add the changes as a separate commit with fix #XXXXXX mention.

Reply:
@sidewayss please address the comment.